src/cookies.njk

```diff
@@ -33,6 +33,10 @@ layout: layout-single-page.njk
 
       <p class="govuk-body">We only set cookies when JavaScript is running in your browser and you’ve accepted them. If you choose not to run Javascript, the information on this page will not apply to you.</p>
 
+      <!--[if lt IE 9]>
+        <p>If you use an older browser, such as Internet Explorer 8 or earlier, the cookies we use will not affect you.</p>
```

Comment:
This is currently unstyled, I think it needs the `govuk-body` class:

```suggestion
        <p class="govuk-body">If you use an older browser, such as Internet Explorer 8 or earlier, the cookies we use will not affect you.</p>
```

Reply:
No comments from me, assume this is because of the 'if' conditional?